composer.json

```diff
@@ -13,7 +13,8 @@
     "require-dev": {
         "php-vcr/phpunit-testlistener-vcr": "^1.1",
         "jakub-onderka/php-parallel-lint": "^0.9.2",
-        "jakub-onderka/php-console-highlighter": "^0.3.2"
+        "jakub-onderka/php-console-highlighter": "^0.3.2",
+        "phpunit/phpunit": "~4.8 || ~5.0"
```

Comment:
Not a fan of this, I've had conflicts with dependencies I've considered integrating with the SDK before. Whatever can safely stay outside of composer should. (I realise that's a bit of a pain in a number of ways but pain > dependency conflicts)

Instead I'd modify this line to conditionally load a different version of phpunit. https://github.com/contentful/contentful.php/pull/69/files#diff-354f30a63fb0907d4ad57269548329e3L21